src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { FaHeart, FaUser, FaShoppingCart } from "react-icons/fa"
+import './Navbar.css';
+import { useCart } from '../../context/cart-context';
+
+const Navbar = () => {
+    const { cartItem, wishlist } = useCart();
+    return (
+        <nav className="navbar" id="header">
+            <div className="navbar-title">
+                <a href="index.html">
+                    JEWELORY
+                </a>
```

Comment:
you may use `Link` from react-router-dom, also you may check the spelling `jewellery`